spec/models/season/phase_switcher_spec.rb

```diff
@@ -4,10 +4,32 @@
 context 'when switching phases' do
   subject { Season.current }
 
+  describe 'destined' do
+    it 'sends the params to a phase setting method' do
+      phase = :fake_application_phase
+      Season::PhaseSwitcher.destined(phase)
+      expect(subject).to_not be_nil
+    end
+
+    it 'addresses the requested phase' do
+      wrong_phase = :fake_proposals_phase
```

Comment:
I don't understand this test. Why is it the wrong phase?

Also, see below for making use of `expect { }.to change { … }`.